README.rst

```diff
@@ -1,61 +1,95 @@
+#########################
 frontend-component-footer
-=========================
+#########################
 
-|Build Status| |Coveralls| |npm_version| |npm_downloads| |license|
-|semantic-release|
+|Build Status| |Codecov| |npm_version| |npm_downloads| |license| |semantic-release|
 
-frontend-component-footer is a library containing a site footer
-component for use when building Open edX frontend applications.
+********
+Overview
+********
 
-Usage
------
+A generic footer for Open edX micro-frontend applications.  It includes a logo with trademark, and an optional language selector dropdown.
```

Comment:
what does "a logo with trademark" mean?

Reply:
I expanded the description - good thought.